CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ endif()
 
 # List of all HAL drivers to be built by default:
 set(IREE_ALL_HAL_DRIVERS
+  Cuda
```

Comment:
this will cause cuda to be built on android/ios/etc as by default IREE_HAL_DRIVERS_TO_BUILD is 'all' - like on line 127 below it's possible to remove the non-relevant backends and we'll want to remove cuda on APPLE and add an additional case for `if(ANDROID)` to remove cuda as well

(this configuration mechanism needs improvements, but doing the above will at least prevent us from bloating android binaries with cuda stuff :)

Reply:
Thanks, make sense. I removed it for ANDROID and APPLE.